packages/main/src/MessageStrip.ts

```diff
@@ -189,41 +176,43 @@ class MessageStrip extends UI5Element {
 
 	static designClassesMappings() {
 		return {
-			"Information": "ui5-message-strip-root--info",
-			"Positive": "ui5-message-strip-root--positive",
-			"Negative": "ui5-message-strip-root--negative",
-			"Warning": "ui5-message-strip-root--warning",
+			Information: "ui5-message-strip-root--info",
```

Comment:
Maybe designClassesMappings and iconMappins could be exported into two enums.

Also for designAnnouncementMappings we can create new type ```Record<MessageStripDesign, string>;``` which will be used as return type of the function.

With these types we can ensure that every key of the enum is handled correctly.